ol3-viewer/src/ome/ol3/Viewer.js

```diff
@@ -697,9 +697,11 @@ ome.ol3.Viewer.prototype.addRegions = function(options) {
 	// add a vector layer with the regions
 	if (this.regions_) {
 		this.viewer_.addLayer(new ol.layer.Vector({source : this.regions_}));
-		// enable roi selection by default
+		// enable roi selection by default,
```

Comment:
sure, as a matter of fact -and I have started this already- if I find myself making changes to a file where there are tabs I replace them from now on. this is only relevant for files that reside in ol3-viewer. it's however somewhat tedious work (like with big files) unless I just leave it at the automatic conversion...